iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp

```diff
@@ -216,6 +217,10 @@ void LinalgFusePass::runOnOperation() {
   }
   tilingOptions.tileInterchange = {tileInterchange.begin(),
                                    tileInterchange.end()};
+  if (doIREEDistribution) {
```

Comment:
I dont know if that is going to be enough. The `TileAndDistributeToWorkgroups` uses https://github.com/google/iree/blob/main/iree/compiler/Codegen/Common/DestructiveUpdateUtils.h to convert from a loop form where values are yielded to a form where you use `flow.dispatch.tensor.load/strore`. Without that things might not connect as easily with IREE?
Its great if it works already, but I doubt it does.

Reply:
It's fine because CSE/Canonicalizer will remove those after IREEComprehensiveBufferize. The IREEComprehensiveBufferize itself would create scf.for loop which returns memref even we don't have rewriteLinalgDestructiveUpdate. The CSE patterns are kicked in and address similar issue. I'll double check if it applies to most of cases. It works for fill + matmul + generic op.

Reply:
Thats awesome actually!

Reply:
Good news! I enabled the flag locally and found that we pass all the checks. There are no stack allocation in IREE's tests.

I think we still have some stack allocation in tensorflow integration tests like https://github.com/google/iree/issues/8592

Actually, I don't really think we can remove the alloca ops in this case. All the results of generic ops are needed during the computation, and we drop one of final results at the end. We may want to revisit if the allocation can be deleted before switching to upstream tile+fuse+distribute.